.travis.yml

```diff
@@ -1,47 +1,162 @@
-language: python
-
 matrix:
   include:
-    - python: 3.7
-      env: TOXENV=py37
+    - name: linux-py27
+      language: python
+      python: 2.7
       dist: xenial
-      sudo: true
-    - python: 3.6
-      env: TOXENV=py36
-      dist: trusty
-      sudo: false
-    - python: 3.6
-      env: TOXENV=memorytest36
-      dist: trusty
-      sudo: false
-    - python: 3.6
-      env: TOXENV=coverage-py36
-      dist: trusty
-      sudo: false
-    - python: "pypy3.5"
-      env: TOXENV=pypy3
-      dist: trusty
-      sudo: false
-    - python: 2.7
-      env: TOXENV=py27
+      sudo: required
+      services:
+        - docker
+      env:
+        - TOXENV=py27
+        - PIP=pip
+        - CIBW_BUILD=cp27-*
+    - name: linux-py35
+      language: python
+      python: 3.5
+      dist: xenial
+      sudo: required
+      services:
+        - docker
+      env:
+        - TOXENV=py35
+        - PIP=pip3
+        - CIBW_BUILD=cp35-*
+    - name: linux-py36
+      language: python
+      python: 3.6
+      dist: xenial
+      sudo: required
+      services:
+        - docker
+      env:
+        - TOXENV=py36
+        - PIP=pip3
+        - CIBW_BUILD=cp36-*
+    - name: linux-py37
+      language: python
+      python: 3.7
+      dist: xenial
+      sudo: required
+      services:
+        - docker
+      env:
+        - TOXENV=py37
+        - PIP=pip3
+        - CIBW_BUILD=cp37-*
+    - name: osx-py27
+      os: osx
+      language: generic
+      env:
+        - PATH="/Library/Frameworks/Python.framework/Versions/2.7/bin:$PATH"
+        - TOXENV=py27
+        - PIP=pip
+        - CIBW_BUILD=cp27-*
+        - CIBW_TEST_COMMAND="pushd {project}; tox ; popd"
+        - CIBW_BEFORE_BUILD="${PIP} install -U pip; ${PIP} install -r requirements.txt"
+      install:
+        - ${PIP} install cibuildwheel==0.10.2
+      script:
+        - cibuildwheel --output-dir dist
+    - name: osx-py35
+      os: osx
+      language: generic
+      env:
+        - TOXENV=py35
+        - PIP=pip2
```

Comment:
pip2?

Reply:
Default Travis OSX image does not have pip command per https://github.com/travis-ci/travis-ci/issues/8829 and we need it to install cibuildwheel

Reply:
Oh and it only comes with python 2.7 per https://blog.travis-ci.com/2017-11-21-xcode8-3-default-image-announce out of the box.